help.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: contribute
+title: Getting Help
+---
+
+# Getting Help
```

Comment:
I  broke things into categories and added a few more resources based on our conversation with Jin at the Bazel docs meeting. (The formatting is probably way off in this comment, but I can send it to you in another format): 

# Getting Help

This page lists Bazel resources beyond the documentation and covers how to get support from the Bazel team and  community.

## Search existing material

In addition to the documentation, you can find helpful information by searching:

- [Bazel user group](https://groups.google.com/g/bazel-discuss)
- [Bazel developer group](https://groups.google.com/g/bazel-dev)
- [Bazel blog](https://blog.bazel.build/) 
- [Stack Overflow](https://stackoverflow.com/questions/tagged/bazel)
- [`awesome-bazel` resources](https://github.com/jin/awesome-bazel) 

## Watch videos

There are recordings of Bazel talks at various conferences, such as:
- Bazel’s annual conference, BazelCon:
  - [BazelCon 2020](https://www.youtube.com/playlist?list=PLxNYxgaZ8RseRybXNbopHRv6-wGmFr04n) 
  - [BazelCon 2019](https://www.youtube.com/watch?v=eymphDN7No4&list=PLxNYxgaZ8Rsf-7g43Z8LyXct9ax6egdSj)
  - [BazelCon 2018](https://www.youtube.com/watch?v=DVYRg6b2UBo&list=PLxNYxgaZ8Rsd3Nmvl1W1B4I6nK1674ezp) 
  - [BazelCon 2017](https://www.youtube.com/watch?v=3eFllvz8_0k&list=PLxNYxgaZ8RseY0KmkXQSt0StE71E7yizG) 
- Bazel day on [Google Open Source Live](https://opensourcelive.withgoogle.com/events/bazel) 


## Ask the Bazel community

If there are no existing answers, you can ask the community by:

- Emailing the [Bazel user group](https://groups.google.com/g/bazel-discuss)
- Emailing the [Bazel developer group](https://groups.google.com/g/bazel-dev)
- Asking a question on [Stack Overflow](https://stackoverflow.com/questions/tagged/bazel)
- Chatting with other Bazel contributors on [Slack](https://slack.bazel.build/) 


## File a bug

If you encounter a bug or want to request a feature, file a [GitHub Issue](https://github.com/bazelbuild/bazel/issues).

Reply:
Done.